src/styles/themes/ThemeColors.ts

```diff
@@ -0,0 +1,89 @@
+type Color = string;
+
+type ThemeColors = {
```

Comment:
In styles.ts migration [PR](https://github.com/Expensify/App/pull/27660) we did it a bit differently, instead of defining it we inferred it based on the default theme

Reply:
In case there won't be a default theme, I would rather define one type and use it for all themes, similar to how it's implemented in this PR. @fabioh8010 wdyt?

Reply:
> I would rather define one type and use it for all themes
I agree with this approach